pontoon/administration/templates/admin.html

```diff
@@ -18,22 +18,39 @@
         <div class="icon fas fa-search"></div>
         <input class="table-filter" type="search" autocomplete="off" autofocus placeholder="Filter projects">
       </div>
-      <a class="add button small" href="{{ url('pontoon.admin.project.new') }}">Add new project</a>
-    </menu>
-
-    {{ ProjectList.header() }}
-
-    {% for project in projects %}
-      {% set main_link = url('pontoon.admin.project', project.slug) %}
-      {% set chart_link = url('pontoon.admin.project', project.slug) %}
-      {% set latest_activity = project.get_latest_activity() %}
-      {% set chart = project_stats.get(project.id, {'total': 0}) %}
+      <a class="add button small" href="{{ url('pontoon.admin.project.new') }}">Add new
+        project</a>
+      <button class="button small request-toggle"
+                    id="toggle-projects"
+                    data-show-disabled="{{ 'true' if show_disabled else 'false' }}"
+                    data-text-show-disabled="Disabled projects >"
+                    data-text-show-enabled="< Enabled projects">
```

Comment:
Note that we use [icons](https://github.com/mozilla/pontoon/blob/main/pontoon/teams/static/js/request.js#L15-L16) in the rest of the UI rather than < and > characters.